webpage/language/en.js

```diff
@@ -0,0 +1,251 @@
+			"demo": {
+				"label": "Demo"
+			}
+		},
+		"settings": {
+			"h2": "Settings",
+			"brightness": "Brightness ",
+			"marquee-text": "Marquee Text",
+			"save": "Save",
+			"speed": "Speed ",
+			"leds": "Number of LEDs ",
+			"position": "Position "
+		}
+	},
+
+	"view": {
+		"h1": "View Options",
+		"front": {
+			"h2": "Front",
+			"text": "Changes to the matrix size are only applied when the clock is restarted.",
+			"variant": "Choice of Clock Variant",
+			"de-10-11": "German 10 × 11",
+			"de-10-11-alt": "German 10 × 11 Alternative",
+			"de-10-11-alt-frame": "German 10 × 11 Alternative Frame",
+			"de-10-11-vertical": "German 10 × 11 Vertical",
+			"de-10-11-clock": "German 10 × 11 Clock",
+			"de-10-11-nero": "German 10 × 11 Nero",
+			"de-11-11": "German 11 × 11",
+			"de-11-11-v2": "German 11 × 11 Version 2",
+			"de-11-11-frame": "German 11 × 11 Frame",
+			"de-22-11-weather": "German 10 × 11 Weather",
+			"de-16-18": "German 16 × 18",
+			"en-10-11": "English 10 × 11",
+			"nl-10-11": "Dutch 10 × 11",
+			"invert-min-dir": "Invert Minute Counting Direction",
+			"mirror-vertically": "Mirror Word Clock Vertically",
+			"mirror-horizontally": "Mirror Word Clock Horizontally"
+		},
+		"language": {
+			"h2": "Language",
+			"hide-it-is": "Hide “It Is”",
+			"at-nine-fifteen": "Um 9:15 ist es ...",
+			"quarter-past-nine": "viertel nach neun",
+			"quarter-nine": "viertel zehn",
+			"at-nine-twenty": "Um 9:20 ist es ...",
+			"twenty-past-nine": "zwanzig nach neun",
+			"ten-before-half-past-ten": "zehn vor halb zehn",
+			"at-nine-fourty": "Um 9:40 ist es...",
+			"twenty-before-ten": "zwanzig vor zehn",
+			"ten-after-half-past-ten": "zehn nach halb zehn",
+			"at-nine-fourtyfive-german": "Um 9:45 ist es...",
+			"quarter-to-ten": "viertel vor zehn",
+			"three-quarter-ten": "dreiviertel zehn",
+			"at-nine-fourtyfive-english": "On 9:45 it is ...",
+			"quarter-to-ten-english": "Quarter to ten",
+			"q-quarter-to-ten-english": "A quarter to ten"
+		},
+		"minutes": {
+			"h2": "Minutes",
+			"mode": "Minutes View Mode",
+			"off": "Off",
+			"normal-4-leds": "Normal (4 LEDs)",
+			"normal-7-leds": "Normal (7 LEDs)",
+			"edges": "Edges",
+			"in-words": "In Words"
+		},
+		"seconds": {
+			"h2": "Seconds",
+			"in-border": "Seconds in the Border",
+			"off": "Off",
+			"point": "Dot",
+			"sector": "Sector",
+			"revolving-sector": "Revolving Sector"
+		},
+		"boot": {
+			"h2": "Boot",
+			"help": "What should happen on restart?",
+			"blink-leds": "Blink All LEDs at the Same Time",
+			"iterate-leds": "Iterate All LEDs One by One",
+			"wifi-symbol": "Show the WiFi Symbol While Searching For a Network",
+			"ip-address": "Show IP Address"
+		}
+	},
+
+	"settings": {
+		"h1": "Settings",
+		"status": {
+			"h2": "Status",
+			"label": "Connection Status",
+			"connect": "Connect"
+		},
+		"led-type": {
+			"h2": "LED Type",
+			"help": "Hier können Sie den Typ des verwendeten LED-Streifens (WS2812 oder SK6812) einstellen.\n" +
```

Comment:
I left some long texts in German that one can translate later.

Reply:
Hey, maybe we can reverence the wiki on GitHub there ?